### DISCUSSION TITLE
Artifacts in Maven Central

### DISCUSSION THREAD

**Discussion by profhenry** [yoheimuta/protolint#412] [General]
We are consuming protolint via maven.
The latest artifact available via maven central is 0.47.6 (https://mvnrepository.com/artifact/io.github.yoheimuta/protoc-gen-protolint)
The latest release in github is 0.49.8.

Are there plans to still deploy the latest releases also to maven central?

**Comment by yoheimuta** [comment]
> Are there plans to still deploy the latest releases also to maven central?

Yes. We should deploy the latest release to maven central, but due to the manual release process, the current artifact is not the latest.
Thank you for letting me know.

If you need the latest, I'm publishing it in a bit. Or, I'll do it after merging your PRs.

**Comment by yoheimuta** [comment]
@profhenry I've just released the latest v0.50.2. See https://central.sonatype.com/artifact/io.github.yoheimuta/protoc-gen-protolint.

**Comment by profhenry** [comment]
@yoheimuta Thanks for publishing 0.50.2 to maven.
When integrating this in our projects we detected #419. :-/
